### ISSUE TITLE
Is it possible to update orator to latest cleo version (0.8.1) ?

### ISSUE THREAD

**Issue by BastienGermond** [sdispater/orator#379]
This is also related in a way to #347, my specific use case is a project using Orator latest version (0.9.9) and I would like to also use Cleo (0.8.1) however right know due to orator I can only use Cleo 0.6.8 :cry:

**Comment by josephmancuso** [comment]
I don't think this project will receive any updates as of late until sdispater is done with his other projects like poetry. 

I have been working on a new ORM designed to be largely a drop in replacement of orator which has all the latest packages. It's a complete rewrite from scratch. 

I wouldn't throw it into production yet but it's here if you want to keep an eye out for it: https://github.com/masoniteframework/orm

**Comment by BastienGermond** [comment]
I'm already watching it ;), so I guess I will migrate to masonite-orm. Thank you for your quick answer.

**Comment by josephmancuso** [comment]
@BastienGermond its still in mid to late beta so I would caution putting it into a production level application. I am currently writing documentation for it. Although you could probably get away with reading the orator docs because its so similar (all the migrations run to the T between Orator and Masonite ORM).

If you just need something up to date and wouldn't be afraid to wait a few days to a week for a feature to be created that you need then by all means, migrate :).

Theres a few features off the top of my head we haven't implemented yet. mainly more advanced features:

* nested eager loading
* has many polymorphic relationships
* inserting related records using associate methods
* No global level transactions 

These are all things we are working on but have not completed as of this comment

**Comment by BastienGermond** [comment]
Sounds like Christmas :D, I am looking forward to it
